.rubocop.yml

```diff
@@ -181,3 +181,4 @@ Style/Next:
 # Keep things fitting nicely on github
 Metrics/LineLength:
   Max: 116
+  AllowURI: true
```

Comment:
@gaganawhad @benhutton How about getting this added to the dg project at some point?